modules/custom/moj_resources/src/CategoryFeaturedContentApiClass.php

```diff
@@ -139,19 +139,27 @@ private function extractSeriesIdsFrom($nodes)
       $seriesIds[] = $n->field_moj_series->target_id;
     }
 
-    return $seriesIds;
+    return array_unique($seriesIds);
```

Comment:
Moved to the function rather than being run immediately on the results of calling this function.